modules/apps/commerce/commerce-checkout-web/src/main/java/com/liferay/commerce/checkout/web/internal/constants/CommerceCheckoutFDSNames.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.commerce.checkout.web.internal.constants;
+
+import com.liferay.commerce.constants.CommercePortletKeys;
+
+/**
+ * @author Luca Pellizzon
+ */
+public class CommerceCheckoutFDSNames {
+
+	public static final String DELIVERY_GROUP =
```

Comment:
usually, in other places, we use the plural version (like in CommerceAccountFDSNames.java or CommerceAddressFDSNames.java)